### DISCUSSION TITLE
send command to a running ffmpeg instance

### DISCUSSION THREAD

**Discussion by medbenzekri** [kokorin/Jaffree#302] [Q&A] [answered]
does Jafree support support sending command to a running ffmpeg instance like in ffmpeg [docs](https://ffmpeg.org/ffplay-all.html#toc-Changing-options-at-runtime-with-a-command) where you can send command to a filter

**Comment by kokorin** [comment, marked as answer]
Sorry for late reply. I checked documentation and unfortunately didn't find any example how to change options at runtime. Could you please give an example?

**Reply by medbenzekri** [reply]
no worries , so as example  when running ffmpeg in cli you can press "c" and it will give you a prompt to either seek to a position or send a command to a filter the only example i could find so far [example](https://stackoverflow.com/questions/33215321/send-seek-command-to-running-ffmpeg-instance/72211485#72211485) and they have also talked about it in this [issue](https://github.com/bramp/ffmpeg-cli-wrapper/issues/13) 
so basically what i want is a way to access process STDIN so i can send my commands while ffmpeg is running 
**Note**: i have tried it cli and its working perfectly 
command that i have tried  : `all -1  out_w 100` this will crop the width to 100 px .

**Reply by kokorin** [reply]
Got it, thanks. It's possible to do what you need but with slight customization. First of all check `FFmpegStopper`. It writes `q` or `qq` chars to ffmpeg stdin to stop ffmpeg. And of course you need to extend `FFmpeg` class and override `createStopper` function.

**Reply by medbenzekri** [reply]
i ll check it out and try to implement it thanks for the guidance and time you put into this .
